schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE DATABASE vet_clinic;
+
+CREATE table animals (
+  id int,
+  name text,
+  date_of_birth date,
+  escape_attempts int,
+  neutered boolean,
+  weight_kg real 
```

Comment:
- [ REQUIRED ] Kindly set the data type of ```name to varchar``` and ```weight_kg to decimal``` to match the requirements.
- [ OPTIONAL ] You can consider marking the id column as primary key, not null and autoincremented, as it ensures uniqueness, data integrity, referential integrity, indexing, and can enhance performance. e.g. ``id int NOT NULL IDENTITY(1,1)`` `` PRIMARY KEY (id)``

Reply:
thanks for the suggestion on the primary key